spyder/preferences/languageserver.py

```diff
@@ -917,7 +929,7 @@ def setup_page(self):
         # --- Advanced tab ---
         # Advanced label
         advanced_label = QLabel(
-            _("Please don't modify these values unless "
+            _("<b>Warning</b>: Please don't modify these values unless "
```

Comment:
Its usually better to tell people what *to* do, rather than what *not* to do.

```suggestion
            _("<b>Warning</b>: Only modify these values if "
```